### ISSUE TITLE
When using zsh over ssh, there are no top-level completions

### ISSUE THREAD

**Issue by zheng** [warpdotdev/Warp#1135]
### Describe the bug

```text
It's not able to complete commands or built-ins

e.g. p<TAB> should show pwd
```

**Comment by michbsd** [comment]
+1

**Comment by tblxdezhu** [comment]
+1

**Comment by 1c3t3a** [comment]
+1

**Comment by szgupta** [comment]
Hey folks, this was fixed in `v0.2022.09.27.08.11.stable_00`. Please feel free to re-open if this still isn't working for you!
